src/v2/guide/class-and-style.md

```diff
@@ -91,69 +91,69 @@ data: {
 }
 ```
 
-Which will render:
+Lo cual va mostrar:
 
 ``` html
 <div class="active text-danger"></div>
 ```
 
-If you would like to also toggle a class in the list conditionally, you can do it with a ternary expression:
+Si también quieres puedes cambiar una lista de clases condicionalmente, lo puedes hacer con una expresión ternaria:
 
 ``` html
 <div v-bind:class="[isActive ? activeClass : '', errorClass]"></div>
 ```
 
-This will always apply `errorClass`, but will only apply `activeClass` when `isActive` is `true`.
+Esto siempre se aplicará`errorClass`, pero solo aplicará `activeClass` cuando `isActive` es `true`.
```

Comment:
Add a single space after the first `aplicará`.
Also sounds better: `Esto siempre aplicará…` remove the `se`.